packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/mocks.tsx

```diff
@@ -38,19 +50,44 @@ export function createMockWebIntegration({
 }: { redirectTo?: string } = {}): ConfirmSignupCodeBaseIntegration {
   return {
     type: IntegrationType.Web,
-    data: { uid: MOCK_UID, redirectTo },
+    data: new WebIntegrationData(
+      new GenericData({
+        uid: MOCK_UID,
+        context: Constants.FX_DESKTOP_V3_CONTEXT,
+        redirect_to: redirectTo,
+      })
+    ),
     getService: () => MozServices.Default,
     getClientId: () => undefined,
     isDesktopRelay: () => false,
   };
+
+  // TBD: Would it be better to use the actual integrations with mocked data representing the query state?
```

Comment:
This sounds like a good idea to make sure that we're running tests with a valid integration.